.gitignore

```diff
@@ -66,7 +66,7 @@ bower_components
 build/Release
 
 # Dependency directories
-node_modules/
+/node_modules/
```

Comment:
Think this may pull in deps from the badge-maker package too. Maybe we could use a negation pattern here if we end up keeping the dir checked in?